core/templates/pages/topic-editor-page/questions-tab/topic-questions-tab.component.spec.ts

```diff
@@ -0,0 +1,224 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for topic questions tab.
+ */
+
+import { EventEmitter } from '@angular/core';
+import { FocusManagerService } from 'services/stateful/focus-manager.service';
+import { QuestionsListService } from 'services/questions-list.service';
+import { SkillSummary, SkillSummaryBackendDict } from 'domain/skill/skill-summary.model';
+import { Subtopic } from 'domain/topic/subtopic.model';
+import { TestBed } from '@angular/core/testing';
+import { TopicRights } from 'domain/topic/topic-rights.model';
+import { TopicsAndSkillsDashboardBackendApiService, TopicsAndSkillDashboardData } from 'domain/topics_and_skills_dashboard/topics-and-skills-dashboard-backend-api.service';
+
+describe('Topic questions tab', function() {
+  var $rootScope = null;
+  var $scope = null;
+  var $window = null;
+  var TopicEditorStateService = null;
+  var TopicObjectFactory = null;
+  var ctrl = null;
+  var focusManagerService = null;
+  var qls = null;
+  var subtopic1 = null;
+  var topic = null;
+  var topicInitializedEventEmitter = null;
+  var topicReinitializedEventEmitter = null;
+
+  let skillSummaryBackendDict: SkillSummaryBackendDict = {
+    id: '3',
+    description: 'description3',
+    language_code: 'language_code',
+    version: 1,
+    misconception_count: null,
+    worked_examples_count: null,
+    skill_model_created_on: 2,
+    skill_model_last_updated: 3
+  };
+
+  let CategorizedSkillsDict: {
+    test: {
+      uncategorized: [],
+      test: []
+    }
+  };
+
+  let skillIdToRubricsObject = {};
+
+  let UntriagedSkillSummaries: SkillSummary[] = (
```

Comment:
Shouldn't this start with a lowercase letter?

Reply:
DONE